example.py

```diff
@@ -0,0 +1,41 @@
+import os
+import sys
+import gdown
+import zipfile
+
+
+BASE_DIR = os.path.dirname(os.path.realpath(__file__))
+DICOM_FILE = os.path.join(BASE_DIR, 'dicom.zip')
+DICOM_ZIP_URL = 'https://drive.google.com/uc?id=1rvyplB7gEoJ-8kzSDuYJg-Dg_NZY6Pr5'
+DICOM_DEST_DIRECTORY = os.path.join(BASE_DIR, 'example')
+
+
+def check_if_destination_directory_exists():
```

Comment:
"check" in func name indicates bool return type, it may be a good idea to clearly indicate the lack of return value
```suggestion
def check_if_destination_directory_exists() -> None:
```